recipes-extended/onl/onl_git.bb

```diff
@@ -40,7 +40,7 @@ SRC_URI = "${URI_ONL};name=onl \
            file://0002-Add-rest-lpmode-code-to-sfpi_control-set-get-api.patch \
            file://0003-Fix-fan-direction-api.patch \
            file://0001-accton-as4630-54pe-Avoid-undefined-behaviour.patch \
-           file://0001-onl-as5835-don-t-ignore-PSU2_AC_PMBUS_NODE.patch \
```

Comment:
Why are you deleting this patch?

Reply:
Sorry, that should of course have been in the commit message. It's a bitch to review in this form, isn't it?

Reply:
Not quite sure what you mean, unless you mean the inability to be able to review commit messages.

I mean I know exactly why you deleted the patch, but you should explain that in your commit message. In contrast to the upstream PR, where they snuck in the fix without mentioning it anywhere.

Reply:
You are right, that's why I amended the commit message to say as much.